frontend/src/text_resources.js

```diff
@@ -251,6 +252,7 @@ let LOCALIZE = new LocalizedStrings({
     //Status Page
     statusPage: {
       title: "Statut de OEC",
+      logo: "Thunder CAT Logo",
```

Comment:
Could you please change this one for "_Logo Thunder CAT_"? This is the French way to say that.